presto-tests/src/test/java/com/facebook/presto/tests/TestTpchLocalStats.java

```diff
@@ -237,4 +237,12 @@ public void testDateComparisons()
                 checks -> checks
                         .estimate(OUTPUT_ROW_COUNT, defaultTolerance()));
     }
+
+    @Test
+    public void testVarcharComparisons()
```

Comment:
I didn't. `LiteralInterpreter.toExpression` could be tested (probably should), but such a test wouldn't be even remotely related to stats work. The stat estimates are so sensitive to and dependent on what conversions a query has already undergone that we need to test stats quite directly and in an integration way.